debug.js

```diff
@@ -0,0 +1,45 @@
+const spawn         = require('child_process').spawn;
+const webdriver     = require('selenium-webdriver');
+const _             = require('lodash/fp');
+
+const os = process.platform  === 'win32' ? 'win' : process.platform;
+
+const fileToDebug = process.argv.splice(2)[0];
```

Comment:
The chrome is throwing an error in console if it cannot find the file, but sure we can check it. We will be able to give quicker feedback for the developer. Cool!